docs/docs/user/intro.md

```diff
@@ -0,0 +1,19 @@
+# Introduction
+
+Users of Moog submit requests for an Antithesis test involving Cardano nodes to
```

Comment:
Perhaps we should provide a bit more details here on who those users are and why they would submit tests through Moog? This is probably described also elsewhere but it feels a bit abrupt :) 

Also: 

```suggestion
Users of Moog submit requests for an Antithesis test involving nodes on a Cardano network to
```